test/unit/image/pixels.js

```diff
@@ -228,7 +228,7 @@ suite('pixels', function() {
       assert.throw(function() {
         let img = myp5.createImage(50, 50);
         img.copy(0, 0, 10, 10, 0, 10, 10);
-      }, 'Signature not supported');
+      });
```

Comment:
This error message is thrown but the detection of the content of the error doesn't work because the [parameter validation also logs a message.](https://github.com/processing/p5.js/pull/3936/files#diff-fecaef9926a3d9ed7f9ac61c8e72e2f0R230). Removing the optional message check works fine.